Runtime/Input/Interfaces/IFocusProvider.cs

```diff
@@ -14,6 +14,8 @@ namespace RealityToolkit.Input.Interfaces
     /// </summary>
     public interface IFocusProvider : ISourceStateHandler, IInputServiceModule
     {
+        public static string MissingFocusProviderMessage = $"Couldn't find a valid {nameof(IFocusProvider)} configured for the {nameof(IInputService)}!\nPlease ensure that at least a Focus Provider has been registered with the {nameof(IInputService)}";
```

Comment:
I think this is okay for now but not really happy with this being here. It's weird to have an error message specified in an interface where we don't know anyhting about the implementation of said interface. It should rather be the input service complaining that there is no focus provider and not the other way.